string object_class
---
bool success
geometry_msgs/PoseStamped pose